.github/workflows/update_knapsack_report.yml

```diff
@@ -0,0 +1,69 @@
+name: Update Knapsack Report
+
+# This workflow should be scheduled at certain intervals
+on:
+ schedule:
+     - cron: '0 5 31 2 *'    
+# The above cron does not run. Replace with the wanted periodicity.
+# For example, the following configuration would schedule it every 3 months
+#    - cron: '0 0 1 */3 *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    services:
+      db:
+        image: postgres:9.4
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    
+    env:
+      RAILS_ENV: test
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 3.1
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 3.1
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Ruby gems
+      run: |
+        gem install bundler
+        bundle install --jobs 4 --retry 3
+    - name: Setup Database
+      run: bundle exec rails db:create db:migrate
+    - name: Update Knapsack Report
+      run: KNAPSACK=true KNAPSACK_GENERATE_REPORT=true bundle exec rspec
+    - name: Commit files
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+        git add knapsack_rspec_report.json
+        git commit -m "Update knapsack_rspec_report.json"
+    - name: Push changes
+      run: git push origin HEAD:update-knapsack-report
+    - name: Initialize Pull Request
+      uses: peter-evans/create-pull-request@v3
```

Comment:
hmmm maybe we should fork this repo? I don't like using others' actions

Reply:
I'll try to do it but It's looks like an oficial repo isn't it ?
https://github.com/marketplace/actions/create-pull-request

Reply:
no it isn't. That's an open marketplace. The repo is https://github.com/peter-evans/create-pull-request so the action was created by some Peter Evans

Reply:
another option is just doing a `curl` request to open the PR using the [Github API](https://docs.github.com/en/free-pro-team@latest/rest/pulls/pulls?apiVersion=2022-11-28#create-a-pull-request)?

something like:

```curl
curl -L \
  -X POST \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer <YOUR-TOKEN>" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/repos/OWNER/REPO/pulls \
  -d '{"title":"Amazing new feature","body":"Please pull these awesome changes in!","head":"octocat:new-feature","base":"master"}'
```